Libraries/test/TestExecutableServerlessApp/SourceGenerationSerializationExample.cs

```diff
@@ -22,8 +22,6 @@ public IHttpResult GetPerson(ILambdaContext context)
         }
     }
 
-    [JsonSerializable(typeof(APIGatewayHttpApiV2ProxyRequest))]
-    [JsonSerializable(typeof(APIGatewayHttpApiV2ProxyResponse))]
```

Comment:
Since the lambda function returns an `IHttpResult`, it does not need to serialize these types.